Mathlib/CategoryTheory/EssentialImage.lean

```diff
@@ -164,6 +164,14 @@ lemma essSurj_of_comp_fully_faithful (F : C ⥤ D) (G : D ⥤ E) [(F ⋙ G).EssS
     [G.Faithful] [G.Full] : F.EssSurj where
   mem_essImage X := ⟨_, ⟨G.preimageIso ((F ⋙ G).objObjPreimageIso (G.obj X))⟩⟩
 
+variable {F} {X : E}
+
+/-- Pre-composing by an essentially surjective functor doesn't change the essential image. -/
+@[simp] lemma essImage_comp_of_essSurj : (F ⋙ G).essImage X ↔ G.essImage X where
```

Comment:
Could be nice to also (instead?) have `(F ⋙ G).essImage = G.essImage`, cf `essImage_eq_of_natIso`